src/components/reducers/product-reducer.js

```diff
@@ -0,0 +1,20 @@
+import { defaultFilters } from "../../contexts/product-context"
```

Comment:
actually I'm using the default filter in the product context to set the initial state in the `useReducer`. To avoid writing the same default filter state again for clear all I am importing it here and using it